<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PieChart</class>
 <widget class="QWidget" name="PieChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PieChart</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/images/piechart.ico</normaloff>:/images/piechart.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Chart</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QtCharts::QChartView" name="chartView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>372</width>
            <height>785</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Basic</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="idEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Title</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="titleEdit">
                <property name="text">
                 <string>Simple Pie Chart</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Misc</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="7" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Animation</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QComboBox" name="animationComboBox">
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <item>
                 <property name="text">
                  <string>No Animation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Grid/Axis Animation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Series Animation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All Animations</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Theme</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QComboBox" name="themeComboBox">
                <item>
                 <property name="text">
                  <string>Light</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Blue Cerulean</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dark</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Brown Sand</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Blue NCS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>High Contrast</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Blue Icy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Qt</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QCheckBox" name="antialiasingCheckBox">
                <property name="text">
                 <string>Anti-Aliasing</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Horizontal Position</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Vertical Position</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Size Factor</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Start Angle</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>End Angle</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Hole Size</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QCheckBox" name="explodeSliceCheckBox">
                <property name="text">
                 <string>Explode Slice On Hover</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="hPosSpinBox">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="vPosSpinBox">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.500000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sizeFactorSpinBox">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.700000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="startAngleSpinBox">
                <property name="minimum">
                 <double>-360.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>1.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QDoubleSpinBox" name="endAngleSpinBox">
                <property name="minimum">
                 <double>-360.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>1.000000000000000</double>
                </property>
                <property name="value">
                 <double>360.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QDoubleSpinBox" name="holeSizeSpinBox">
                <property name="maximum">
                 <double>0.700000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Legend</string>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QCheckBox" name="legendShowCheckBox">
                <property name="text">
                 <string>Show Legends</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Position</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="legendPositionComboBox">
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <item>
                 <property name="text">
                  <string>Top</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Right</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bottom</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Left</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Detached</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QGroupBox" name="legendDetachedParamGroup">
                <property name="title">
                 <string>Detached Parameters</string>
                </property>
                <layout class="QFormLayout" name="formLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>HPos</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="legendHPosSpinBox">
                   <property name="maximum">
                    <double>1920.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>150.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>VPos</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="legendVPosSpinBox">
                   <property name="maximum">
                    <double>1920.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>150.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Width</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QDoubleSpinBox" name="legendWidthSpinBox">
                   <property name="maximum">
                    <double>1920.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>150.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Height</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QDoubleSpinBox" name="legendHeightSpinBox">
                   <property name="maximum">
                    <double>1920.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>75.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Font</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QFontComboBox" name="legendFontComboBox"/>
              </item>
              <item row="6" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="legendBoldCheckBox">
                  <property name="toolTip">
                   <string>Bold</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../icons.qrc">
                    <normaloff>:/images/format-text-bold.png</normaloff>:/images/format-text-bold.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="legendItalicCheckBox">
                  <property name="toolTip">
                   <string>Italic</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../icons.qrc">
                    <normaloff>:/images/format-text-italic.png</normaloff>:/images/format-text-italic.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="legendUnderlineCheckBox">
                  <property name="toolTip">
                   <string>Underline</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../icons.qrc">
                    <normaloff>:/images/format-text-underline.png</normaloff>:/images/format-text-underline.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="6" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="topMargin">
                 <number>4</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Font Size</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="legendFontSizeSpinBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                  <property name="value">
                   <double>8.250000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Labels</string>
             </property>
             <layout class="QFormLayout" name="formLayout_5">
              <item row="0" column="0">
               <widget class="QCheckBox" name="showLabelsCheckBox">
                <property name="text">
                 <string>Show Labels</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="toolTip">
                 <string>Only applicable when position is set to &quot;Outside&quot;</string>
                </property>
                <property name="text">
                 <string>Label Arm Length</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="labelArmLengthSpinBox">
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>99.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>0.200000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Position</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="labelPosComboBox">
                <item>
                 <property name="text">
                  <string>Outside</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Inside Horizontal</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Inside Tangential</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Inside Normal</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
